Alignment/MillePedeAlignmentAlgorithm/plugins/MillePedeDQMModule.cc

```diff
@@ -196,10 +196,10 @@ void MillePedeDQMModule ::fillExpertHistos() {
 }
 
 void MillePedeDQMModule ::fillExpertHisto(MonitorElement* histo,
-                                          const std::array<double, 6>& cut,
-                                          const std::array<double, 6>& sigCut,
-                                          const std::array<double, 6>& maxMoveCut,
-                                          const std::array<double, 6>& maxErrorCut,
+                                          const std::array<double, 8>& cut,
+                                          const std::array<double, 8>& sigCut,
+                                          const std::array<double, 8>& maxMoveCut,
+                                          const std::array<double, 8>& maxErrorCut,
                                           const std::array<double, 6>& obs,
```

Comment:
Why still "6"?

Reply:
Because the `fillExpertHisto` method will only take care of the low granularity result, which has 6 different structures. We will add a new method in an upcoming PR to take care of the high granularity result.